licenses/jackson-module-paranamer-2.11.3.jar.sha1

```diff
@@ -0,0 +1 @@
+2f1e98567b56705c026cbc9a808f969b64eead98
```

Comment:
This file should be ignored

Reply:
So are we supposed to ignore all `sha1` file changes under `licenses` directory? If so, I can add it into `.gitignore`